src/psqlgraph/mocks.py

```diff
@@ -1,8 +1,8 @@
 import logging
 
-from psqlgraph_test_utils import mocks
+from psqlgraph import hydrator
 
 logger = logging.getLogger(__name__)
-logger.warning(f"This module {__name__} is depreciated, please import {mocks.__name__}")
+logger.warning(f"This module {__name__} is depreciated, please import {hydrator.__name__}")
```

Comment:
nit:
typo `deprecated`

More importantly: python `warnings`.
Check it out - it might be worth it.

https://docs.python.org/3.8/library/warnings.html